bugs/mahrud/trace_test.m2

```diff
@@ -0,0 +1,22 @@
+debug Core
```

Comment:
This file is related to Macaulay2, but it doesn't report a bug.  Is it a test?  An experiment?

Reply:
It's a test of the PROFILING option, but not one that could be moved to `Macaulay2/tests` since it requires a build option. Where should I move it?

Reply:
Let's devise a way for it to tell whether the test can be run, so the file can be in `Macaulay2/tests`.  Maybe we could add some things to `version`.

By the way, could it ever fail?  Does it check or verify that any results are correct?  If not, in what sense is it a test?

Reply:
They can fail if the stack trace shows too many things or not enough things, or if it crashes. The profiling feature is still rather experimental. When it is more ironed out I will add better tests, but that would be a future PR.